### ISSUE TITLE
Binding is missing for INSAccessibilityCheckBox

### ISSUE THREAD

**Issue by chamons** [xamarin/xamarin-macios#7079]
Note that bindings are present for the other 18 or so subclasses of NSAccessibility but INSAccessibilityCheckBox is missing, for some reason.

Originally: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/987346
